src/helpers.php

```diff
@@ -56,12 +56,12 @@ function backpack_users_have_email()
     function backpack_avatar_url($user)
     {
         $firstLetter = $user->getAttribute('name') ? mb_substr($user->name, 0, 1, 'UTF-8') : 'A';
-        $placeholder = 'https://placehold.it/160x160/00a65a/ffffff/&text='.$firstLetter;
+        $placeholder = 'https://place-hold.it/160x160/00a65a/ffffff/&bold&fontsize=18&text='.$firstLetter;
```

Comment:
```suggestion
        $placeholder = 'https://place-hold.it/160x160/00a65a/ffffff/&fontsize=40&text='.$firstLetter;
```

I found no bold and font size 40 to be the closest to what we had before:
![2021-01-04 08 39 33](https://user-images.githubusercontent.com/1032474/103536169-669c0b80-4e68-11eb-8e5d-034e79c40ee8.gif)